tests/unit_tests/test_enkf_main.py

```diff
@@ -2,26 +2,9 @@
 from pathlib import Path
 
 import pytest
-from ecl.summary import EclSum
 
-from ert.config import AnalysisConfig, EnsembleConfig, ErtConfig, ModelConfig
-from ert.enkf_main import EnKFMain, create_run_path, sample_prior
-
-
-@pytest.mark.unstable
-def test_ecl_config_creation(minimum_case):
-    assert isinstance(minimum_case.analysisConfig(), AnalysisConfig)
-    assert isinstance(minimum_case.ensembleConfig(), EnsembleConfig)
-
-    with pytest.raises(AssertionError):  # Null pointer!
-        assert isinstance(minimum_case.ensembleConfig().refcase, EclSum)
```

Comment:
Whats the thinking behind removing these tests?

Reply:
They are duplicate, and the `# Null pointer!` is a reference to a time long ago when these were in C